src/dipdup/context.py

```diff
@@ -809,6 +812,9 @@ def __init__(
             handler_config.parent.name if handler_config.parent else 'unknown',
             handler_config.parent._template_values if handler_config.parent else {},
         )
+    
+    def __repr__(self) -> str:
+        return f'<Class {self.__class__.__name__}(handler_config={self.handler_config}, template_values={self.template_values})>'
```

Comment:
Please, leave only the handler name.

Reply:
@droserasprout To get the handler name would be self.template_values._index or just handler_config.parent.name?

Reply:
I think the best format would be `<index>.<handler>`.